core/trino-main/src/main/java/io/trino/server/ServerMainModule.java

```diff
@@ -213,6 +214,8 @@ protected void setup(Binder binder)
             httpServerConfig.setHttp2MaxConcurrentStreams(32 * 1024); // from the default 16K
         });
 
+        newOptionalBinder(binder, Key.get(boolean.class, EnableVirtualThreads.class))
+                .setBinding().toInstance(true);
```

Comment:
I believe this change causes coordinator to become unresponsive. How about limiting this to just workers?

Reply:
Thanks for the update. I still believe that the PR can improve the cluster performance. Looking forward to see this get merged soon.

Reply:
@sajjoseph There is some threading issue in Jetty that also stalls the cluster under HTTP/2